tools/kompat/kompat.go

```diff
@@ -12,7 +12,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package main
```

Comment:
Packages should be directories so if you need to change the package name, you should create a sub-directory